src/classes.py

```diff
@@ -0,0 +1,54 @@
+import sqlalchemy as sa
+
+
+class SQLiteDataBase:
+    def __init__(self, database_file: str):
+        self.db = sa.create_engine(f'sqlite:///{database_file}')
+
+    def get_all_columns(self, table: str) -> '[]':
+        columns = self.db.execute(f'''
+                        PRAGMA table_info('{table}')
```

Comment:
please follow this tutorial https://docs.sqlalchemy.org/en/14/core/tutorial.html#using-textual-sql and update your sql requests.

At this moment it has a security risk called SQL Injection. Please read more herehttps://www.hacksplaining.com/exercises/sql-injection